src/ServiceControl.MultiInstance.AcceptanceTests/SagaAudit/When_sending_saga_audit_to_main_instance.cs

```diff
@@ -22,10 +26,17 @@ public void ConfigSetup()
             var config = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
             var appSettings = (AppSettingsSection)config.GetSection("appSettings");
             appSettings.Settings.Add("ServiceControl/Queue", ServiceControlInstanceName);
+            MimicSettingsOfAnUpgradedOldMainInstanceThatHasNeverHadAuditRetentionSetting(appSettings);
             config.Save();
             ConfigurationManager.RefreshSection("appSettings");
+
+
         }
 
+        static void MimicSettingsOfAnUpgradedOldMainInstanceThatHasNeverHadAuditRetentionSetting(
+            AppSettingsSection appSettings) =>
+            appSettings.Settings.Remove("ServiceControl/AuditRetentionPeriod");
```

Comment:
We might want to store this so that we can restore it after the test. The config file is shared by all of the tests in the same assembly. I _think_ none of the other tests are relying on it being there but it might be a nasty surprise for someone trying to write a test in the future. It will work when run in isolation but will fail when it's run after this (which may not even be deterministic).